qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ReportHistoryPage.java

```diff
@@ -22,7 +22,7 @@ public ReportHistoryPage(Page page) {
 	}
 
 	public RenderDefaultReportPage clickOnViewLink() {
-		waiter.until(ExpectedConditions.visibilityOfElementLocated(VIEW_REPORT));
+		waiter.until(ExpectedConditions.elementToBeSelected(VIEW_REPORT));
```

Comment:
`waiter.until(ExpectedConditions.elementToBeClickable(VIEW_REPORT))`;

Sorry for the mixup..

Reply:
Will  execute whatever ideas just to let you comfirm that they are failing.